scripts/docker/baseDockerfile

```diff
@@ -40,7 +40,7 @@ WORKDIR /home/idflakies/
 RUN \
   wget http://mirrors.ibiblio.org/apache/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz && \
   tar -xzf apache-maven-3.5.4-bin.tar.gz && mv apache-maven-3.5.4/ apache-maven/ && \
-  echo 'export JAVA_HOME=/usr/lib/jvm/java-8-oracle' >> ~/.bashrc && \
+  echo 'export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/' >> ~/.bashrc && \
```

Comment:
Changes in one of the commits was already merged in from a different pull request (e.g., changes here to the baseDockerfile). Is there some way to remove that commit from this pull request? If we don't remove the commit, will it cause issues with the merge, that we already have merged a similar commit in?

Reply:
Ah sorry, I had a bit of a merging chaos on my fork. I think it should not cause any issues, but I just changed the branch to base the commit on iDFlakies' old master.